components/tree/index.zh-CN.md

```diff
@@ -44,7 +44,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/Xh-oWqg9k/Tree.svg
 | selectable | 是否可选中 | boolean | true |  |
 | selectedKeys | （受控）设置选中的树节点 | string\[] | - |  |
 | showIcon | 是否展示 TreeNode title 前的图标，没有默认样式，如设置为 true，需要自行定义图标相关样式 | boolean | false |  |
-| showLine | 是否展示连接线 | boolean \| {showLeafIcon: boolean} | false |  |
+| showLine | 是否展示连接线 | boolean \| {showLeafIcon: boolean; leafIcon?: ReactNode | ((props: AntTreeNodeProps) => ReactNode)} | false |  |
```

Comment:
`leafIcon` is not existed according to https://github.com/ant-design/ant-design/pull/37144#discussion_r950838350

Reply:
Good catch! Just fixed in [#ad19e58](https://github.com/ant-design/ant-design/pull/37144/commits/ad19e58430e2a6811c087dd7df93b190f5b5c8b4)